### PR TITLE
QA: fix sticky position of Gallery block

### PR DESCRIPTION
**Pull request by nahbee10**
### Description

- remove `overflow-hidden` from Gallery container to fix not working sticky position setup for Gallery text 

### Type(s) of changes

- [x] Bug fix
- [ ] New feature
- [ ] Update to an existing feature

### Motivation for PR

QA item from Notion
- [the sticky effect from the live website has been broken](https://www.notion.so/garden3d/It-appears-the-sticky-effect-from-the-live-website-has-been-broken-in-the-preview-link-a1b7d642bfc4402188006e550509bc9a?pvs=4)

### How Has This Been Tested?

- Tested with [preview link](https://sanctu-dot-md7x363fb-sanctucompu.vercel.app/) on desktop browser

### Applicable screenshots

https://www.loom.com/share/5c5a0952ead14564b1840db82f1e3968

### CODE REVIEW

**Review by mokaymokay — COMMENTED** [review]
I think we added this to fix the issue of links overflowing horizontally, can we add `overflow-x-hidden` (maybe to Markdown?) to make sure that issue won't reoccur?

**Comment by nahbee10** [comment]
> I think we added this to fix the issue of links overflowing horizontally, can we add `overflow-x-hidden` (maybe to Markdown?) to make sure that issue won't reoccur?

good point! I implemented this by adding `overflow-x : hidden` to separate CSS file since any Basement overflow classes(hidden, scroll, auto) will disable sticky effect. Here's the [PREVIEW](https://sanctu-dot-hv6ga8wv4-sanctucompu.vercel.app/) for fixed version!

**Comment by mokaymokay** [comment]
Alternatively, we can add it to Markdown as I've suggested:

Screen Shot 2023-02-10 at 5 30 24 PM

**Comment by nahbee10** [comment]
> Alternatively, we can add it to Markdown as I've suggested:
> 
@mokaymokay This doesn't solve the issue! I think the overflow should be prevented from more parent container.
Screen Shot 2023-02-10 at 12 39 20 PM

File: styles/config.scss [hunk 1]
```diff
@@ -19,3 +19,11 @@ $spacing-units: 0rem, 0.25rem, 0.325rem, 0.5rem, 0.625rem, 0.75rem, 0.9rem, 1rem
...
 
 $grid-columns: 8;
+
+$additional-class-definitions: (
+  'overflow-x-visible': (
+    overflow-x: visible,
+  ),
+);
+
+$additional-responsive-classes: (overflow-x-visible);
```
Comment (nahbee10) [inline styles/config.scss:29]: @mokaymokay I ended up with using custom basement definitions!

Comment (mokaymokay) [reply styles/config.scss:29]: nice

**Review by mokaymokay — APPROVED** [review]

**Pull request merged**